practices/java-jdbc/test/java/com/uabc/dowa/jdbc/repository/StudentRepositoryTest.java

```diff
@@ -2,26 +2,59 @@
 
 import org.junit.Test;
 
-import static org.junit.Assert.fail;
+import com.uabc.dowa.jdbc.model.Student;
+import java.sql.SQLException;
+
+import static org.junit.Assert.*;
 
 public class StudentRepositoryTest {
     @Test
-    public void testAddStudent() {
-        fail("Implement me!");
+    public void testAddStudent() throws SQLException {
+
+        StudentRepository repository = new StudentRepository();
+
+        Student student = new Student();
+        student.setFirstName("Ivan");
+        student.setLastName("Duarte");
+
+        repository.persistStudent(student);
     }
 
     @Test
-    public void testFindAllStudents() {
-        fail("Implement me!");
+    public void testFindAllStudents() throws SQLException {
+        StudentRepository repository = new StudentRepository();
+
+        assertEquals(repository.findAllStudents().size(), 0);
```

Comment:
I recommend you to add a student and then assert on 'repository.findAllStudents()' not being empty.